### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in Caregivers end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7303]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. While auditing the `/caregivers` app folder, I wasn't able to determine if coverage was adequate. I'd like the front-end engineering team to review this application, and add end-to-end tests as needed. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/caregivers/tests`
* `$ yarn test:e2e src/applications/caregivers/tests`

## Acceptance Criteria

**Definition of done:**
- [x] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [x] axe checks are run for every unique page or view
- [x] axe checks are run for hidden content like modal windows, accordions
- [x] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [x] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by ala-yna** [comment]
@jenstrickland - Can this be closed?

**Comment by jenstrickland** [comment]
@ala-yna The items under Definition of Done above are not checked off. Have the developers completed all those steps?

**Comment by jenstrickland** [comment]
@LinnJS @ala-yna have the acceptance criteria items listed above been completed? cc @1Copenut

**Comment by LinnJS** [comment]
Since we are using the VSP `FormBuilder` we can use the awesome Cypress utility function that VSP provided recently.

I knew they ran AxeChecks on the form pages, but was not sure about the container files like the `introduction` and `confirmation` pages.

Went and ran the test and looks like they test every page on initial render gathered some screen shots as well.

![Screen Shot 2020-08-18 at 1.21.25 PM.png](https://images.zenhubusercontent.com/5dcf313b98e0b800015f2353/6baa3008-e5dc-421d-b27d-51edd76d2156

![Screen Shot 2020-08-18 at 1.21.32 PM.png](https://images.zenhubusercontent.com/5dcf313b98e0b800015f2353/e0c499ab-1e58-4d85-9a4d-e782716c9ec3))

**Comment by 1Copenut** [comment]
@LinnJS I appreciate you dropping in the screen shots. I forgot the Cypress form tests run `axeCheck()` automatically now. Having these checks in place is a very good thing.
